cetmix_tower_server_queue/models/cx_tower_file.py

```
@@ -0,0 +1,148 @@
+            self._do_upload(raise_error=raise_error)
+
+    def download(self, raise_error=False):
+        """
+        Trigger asynchronous download via job queue.
+        """
+
+        # Check if the file is already being processed
+        processed_files = self.filtered(lambda file: file.is_being_processed)
+        if processed_files:
+            if raise_error:
+                raise UserError(_("The following files have already been processed: %(name)s", name=",".join(processed_files.mapped("name"))))
```

Ruff (0.8.2)

Line too long (142 > 88)

(E501)